src/App.tsx

```diff
@@ -125,7 +125,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     margin: 'auto',
     minHeight: '100vh',
     '& .MuiCircularProgress-svg': {
-      color: theme.palette.ClrIconFillProductive,
+      color: theme.palette.TwClrIcnBrand,
```

Comment:
@hannah-cai @rachel-tf-design Does this seem like a suitable replacement? It is `#007DF2` (old) vs `#2C8658` (new) and is for the initial large app loading spinner.

Reply:
@charlie-tf Yes, that's the correct replacement